octodns_powerdns/__init__.py

```diff
@@ -662,6 +663,8 @@ def __init__(
         host,
         api_key,
         port=8081,
+        scheme="http",
+        ssl_verify=False,
```

Comment:
These are already parameters, they're defined in `PowerDnsBaseProvider` and come through here as part of `kwargs`.